### DISCUSSION TITLE
Timeouts/Resilience for loudness-analysis?

### DISCUSSION THREAD

**Discussion by RandomUser345** [RandomNinjaAtk/docker-lidarr-extended#19] [Ideas]
I'm quite a noob when it comes to bash-scripting so I don't know if this is even feasible, but on my hardware (Raspberry Pi 4 with Ubuntu 20.04) I sometimes download files from deezer/tidal which cause the loudness-analysis to crash, bringing down the whole box. Is there a way to make this more resilient? Since I cant login anymore to the machine and have to reset it I'm not able to get to much info, I just see the last lines in the docker logs after the restart and its always on the loudness-check.
Fortunately the same files won't be downloaded again since they are already in the list, still this can be quite annoying if the downloader is idling for a day before I see the issue.

**Comment by RandomNinjaAtk** [comment]
I've thought about making the feature optional, this may provide the push to make that change.

**Comment by RandomNinjaAtk** [comment]
Latest version should give you some options to disable it, see the updated readme

**Reply by RandomUser345** [reply]
Works like a charm! Thank you!
